todo_rxbloc_app/lib/feature_todos_list/bloc/todos_list_manage_bloc.dart

```diff
@@ -78,4 +104,34 @@ class TodosListManageBloc extends $TodosListManageBloc {
 
   @override
   Stream<bool> _mapToIsLoadingState() => loadingState;
+
+  @override
+  Stream<OptionsMenuEnum> _mapToMarkTodosCompleteState() => Rx.merge([
+        Rx.combineLatest2<OptionsMenuEnum, List<TodoEntity>, ReturnValues>(
+            _$markAllEvent,
+            todosList,
+            (menu, todos) => ReturnValues(todos, menu)).switchMap((value) {
```

Comment:
Considering there is no async operation I believe you do not need to use the `switchMap` operator. You can either use the map operator or to move the code to the combineLatest2's third argument